packages/mantine/package.json

```diff
@@ -30,11 +30,13 @@
     "dependencies": {
         "@coveord/plasma-react-icons": "workspace:*",
         "@coveord/plasma-tokens": "workspace:*",
+        "@mantine/carousel": "5.6.4",
```

Comment:
We put the mantine stuff in the devDependencies and peerDependencies to allow us to have a different version in packages that install plasma-mantine